bt buổi 10/main.js

```diff
@@ -0,0 +1,96 @@
+function getMultiplication() {
+    for(let i = 0; i < 10; i++) {
+        console.log(6 * i)
+    }
+}
+
+// Bài 1
+const printMultiplication = number => {
+    for(let i = 0; i < 10; i++) {
+        console.log(6 * i)
```

Comment:
ở đâu ra `6 * i` đây, nhập `number` vào để làm gì?
Nhập `number` bằng bao nhiêu thì in ra bảng cửu chương ấy chứ